backend/src/main/java/com/github/waymaker7/capstone/BlogsModel.java

```diff
@@ -0,0 +1,17 @@
+package com.github.waymaker7.capstone;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/Blogs")
+
+public class BlogsModel {
+    @GetMapping
+    public List<Blogs> getTasks(){
+        return null;
+    }
+}
```

Comment:
what is this for? 
You already got a Controller. 
with exactly this methods in `BlogsController`.

Reply:
i have deleted it